pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,16 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+// backoff is the underlying backoff struct we use to prevent thundering herd drama, esp when this runs in a controller.
+var backoff = &URLBackoff{Backoff: util.NewBackOff(1*time.Second, 120*time.Second)}
```

Comment:
How does this sound: 

1. Remove global from this location and place only into the urlbackoff utility
2. Have the urlbackoff utility only read in response codes from the requests.
3. Have the Throttle.Accept call take in a Duration variable from a urlbackoff function that used the "previous response codes".  
4. Modify Throttle.Accept to use the new Duration.